### ISSUE TITLE
Postman just hang, can't force quit on osx

### ISSUE THREAD

**Issue by rawaludin** [a85/POSTMan-Chrome-Extension#3379]
## App Details:
Can't open postman. It hang.

```
Postman for Mac
Version 5.1.3
OS X 10.12.2 / x64
```

## Issue Report:
I don't know how to reproduce this, but I believe this is because large payload. My postman currently hang and force quite can't close it. I tried open activity monitor, but couldn't find process named postman. I tried `ps aux | grep postman` found nothing. The postman app keep running

<img width="845" alt="screen shot 2017-08-13 at 9 55 30 am" src="https://user-images.githubusercontent.com/1274849/29245748-1c620596-800e-11e7-9406-7807c831f13f.png">

I tried to replace the postman app, it said file in use:

![screen shot 2017-08-13 at 10 00 23 am](https://user-images.githubusercontent.com/1274849/29245755-40483b7e-800e-11e7-843b-892e7ad1044b.png)

I tried `lsof | grep postman` found nothing.
I also tried `sudo fs_usage | grep postman` found nothing too.

I tried to shutdown my mac too. All app closed, except postman. Which prevent me to shutdown my mac normally. 

The only solution I find is to force shutdown my mac :(

**Comment by kamalaknn** [comment]
Hey @rawaludin 

That is bad. Could you share some more details to help us narrow down on the issue?

Does this happen every time or at random times?
Could you share the size of payload that crashed the app, also the nature of the content (plain text/zip etc)?
Were you able to see this right after you start the app for the first time, or does it happen after you have the app open for a long time?

Thanks.

**Comment by rawaludin** [comment]
Yes, it happen after I open postman for a long time.

**Comment by gerciljunio** [comment]
Same problem here!

**Comment by kamalaknn** [comment]
Hi,

Can you update your app to the latest version of Postman and check if this issue still exists?

**Comment by kamalaknn** [comment]
Closing this due to inactivity

**Comment by rs-renato** [comment]
> Same problem here!

Algo here!!

**Comment by rs-renato** [comment]
> ## App Details:
> Can't open postman. It hang.
> 
> ```
> Postman for Mac
> Version 5.1.3
> OS X 10.12.2 / x64
> ```
> 
> ## Issue Report:
> I don't know how to reproduce this, but I believe this is because large payload. My postman currently hang and force quite can't close it. I tried open activity monitor, but couldn't find process named postman. I tried `ps aux | grep postman` found nothing. The postman app keep running
> 
> <img alt="screen shot 2017-08-13 at 9 55 30 am" width="845" src="https://user-images.githubusercontent.com/1274849/29245748-1c620596-800e-11e7-9406-7807c831f13f.png">
> 
> I tried to replace the postman app, it said file in use:
> 
> ![screen shot 2017-08-13 at 10 00 23 am](https://user-images.githubusercontent.com/1274849/29245755-40483b7e-800e-11e7-843b-892e7ad1044b.png)
> 
> I tried `lsof | grep postman` found nothing.
> I also tried `sudo fs_usage | grep postman` found nothing too.
> 
> I tried to shutdown my mac too. All app closed, except postman. Which prevent me to shutdown my mac normally.
> 
> The only solution I find is to force shutdown my mac :(

exact the same, it was annoying

**Comment by HebeHH** [comment]
This just happened to me after I restarted my mac, it's still hanging now.

**Comment by kamalaknn** [comment]
Hey @HebeHH 

Could you share the application logs from `View -> Developer -> View logs in Finder` with us through `help@postman.com`?

We're working on some performance improvements and would love to understand the workflows that are causing trouble for you.

**Comment by brinal8055** [comment]
> This just happened to me after I restarted my mac, it's still hanging now.

Same issue. What's the next step we can do for force quit.?

**Comment by ranggasi** [comment]
**ps -ef | grep Postman**
get the id
**kill -9 ID**

**Comment by sulemanahsancui** [comment]
same issue nothing works it's stuck and Mac is not shut down due to this.  anyone has found solution how to quit it?

**Comment by afzalIbnSH** [comment]
> **ps -ef | grep Postman** get the id **kill -9 ID**

Did you try this @sulemanahsancui

**Comment by ninadvsane** [comment]
https://support.apple.com/en-in/HT201276

Press these three keys together: Option (or Alt), Command, Esc (Escape).1 Or choose Force Quit from the Apple menu  in the corner of your screen.

How to force quit on Mac: press Option-Command-Esc

A Force Quit window will open. Select the app you want to close, then click Force Quit:

This worked for me.

**Comment by hendisantika** [comment]
> **ps -ef | grep Postman** get the id **kill -9 ID**

Nice info. I try 
```
ps -ef | grep postman
```
lowercase

**Comment by asraful** [comment]
`ps auxww | grep -i 'postman' | grep -v grep | awk '{ print $2 }' | xargs kill -9 `

**Comment by myrzataev** [comment]
killall Postman
helped in my case

**Comment by tristanaustin** [comment]
I realise this is closed, but it's not fixed. I just downloaded the latest version of Postman (10.24.0) and the same thing happened.

Specs:
- Apple M3 Max
- 14.2.1 (23C71)

Context:
- I was seeing "You do not have permission" on a bunch of buttons. Ran a quick google search and saw someone say restarting the app may help.
- Quit the app. The window disappeared, but the Postman app icon was still present in the app switcher when using Command-Tab.
- Initially it was also present in the Force Quit dialog and did not respond to multiple attempts to Force Quit
- Resorted to the command line. Did not die, but after a few attempts it disappeared from the Force Quit dialog, but was still present in the ps listing on the terminal.

To summarise

- Force quit does not work
- sudo kill -9 PID does not work
- sudo killall Postman does not work

I now have two unkillable Postman processes that are only visible in the terminal when running ps -eaf | grep Postman
![UnkillablePostman](https://github.com/postmanlabs/postman-app-support/assets/36911621/dd3b0eaf-6207-48fe-892f-dc1d285aaf15)

**Comment by kamalaknn** [comment]
@tristanaustin Could you help us with some additional information to identify the cause for the issues?

1. Could you share the application logs with us at `help@postman.com`? If you are able to open the app you can find them under `View -> Developer -> View logs in Finder`. If you are not able to get the app to open, you can find the logs at `~/Library/Application\ Support/Postman`
2. Could you share a screenshot of the active files accessed by the process consuming a lot of memory from activity monitor? You can find it under activity monitor here

<img width="971" alt="Screenshot 2024-03-13 at 10 51 46 AM" src="https://github.com/postmanlabs/postman-app-support/assets/8401064/5e87538a-bcef-49ac-8bb3-1d4178ee43da">
